### PR TITLE
Update Core-CLI reference page - Closes 743

### LINKED ISSUE #743: Update `CLI reference`
### Description

Update Lisk Core CLI pallet explanation to be compatible with Lisk Core v4

### PR DESCRIPTION
**Pull request by TalhaMaliktz**
### What was the problem?
Closes #743 

### How was it solved?

Documented all the CLI commands for `lisk-core`

### CODE REVIEW

File: docs/antora/modules/ROOT/pages/core-cli.adoc [hunk 1]
```diff
@@ -0,0 +1,7955 @@
...
+
+=== help
+Displays a summary of all available CLI commands of Lisk Core.
+
+[source,bash]
+----
+Lisk blockchain application platform
+
+VERSION
+  lisk-core/4.0.0-alpha.18 darwin-arm64 node-v16.18.0
```
Comment (Tschakki) [inline docs/antora/modules/ROOT/pages/core-cli.adoc:27]:
```suggestion
  lisk-core/4.0.0 darwin-arm64 node-v16.18.0
```

**Review by Tschakki — APPROVED** [review]
LGTM 👍

File: docs/antora/modules/ROOT/pages/core-cli.adoc [hunk 2]
```diff
@@ -0,0 +1,7955 @@
...
+
+TOPICS
+  block          Commands relating to Lisk Core blocks.
+  blockchain     Commands relating to Lisk Core blockchain data.
+  config         Commands relating to Lisk Core node configuration.
+  endpoint       Invokes the provided endpoint.
+  generator      Disable block generation for given validator address.
+  genesis-block  Creates genesis block file.
+  keys           Return keys corresponding to the given passphrase.
+  node           Commands relating to Lisk Core node.
```
Comment (nagdahimanshu) [inline docs/antora/modules/ROOT/pages/core-cli.adoc:40]: Shouldn't it be `system`?

**Review by nagdahimanshu — APPROVED** [review]
LGTM 👍

**Pull request merged**